plugin/Controller/InputVariableReaderFormatter.php

```diff
@@ -52,7 +52,7 @@ public function formatValue($value, string $type)
 	 */
 	public function formatFloatValue(float $value): string
 	{
-		return number_format_i18n($value, 2);
+		return number_format($value, 2, '.', '');
```

Comment:
That causes problems with the format that I use to enter prices.

example in the screenshots: 

- I add one hundred (100) to search price field. 
- After searching: field formatted one hundred to 100,00 automatically. (usual format for germany)
- If I search again the price increase to ten thousand 10000,00

Reply:
![2024-04-03 09_12_05-WP-Plugin Test – Testinstanz für das W3PM](https://github.com/onOffice-Web-Org/oo-wp-plugin/assets/22166320/fdacc1a2-c3d2-4ee1-9837-b4a86c6b1fc1)

Reply:
![2024-04-03 09_12_29-WP-Plugin Test – Testinstanz für das W3PM](https://github.com/onOffice-Web-Org/oo-wp-plugin/assets/22166320/7f363dc5-c706-48f9-82f2-4e0b122d566f)

Reply:
@andernath I will check it

Reply:
@andernath Currently, this bug occurs for input[type=number] with fields of type float being displayed in frontend. The input display value for input[type="number"] in the frontend is being formatted by parseFloat() method and the value of input[type="number"] will vary depending on the language (Selected in WordPress Backend Site Language) 
![image](https://github.com/onOffice-Web-Org/oo-wp-plugin/assets/104892108/d19b5931-f580-4bfa-8050-dd0144a79878)
**For example:**
- On the first search
-> I add one hundred (100) to the search price field. When going through the parseFloat() method , will return value = 100
-> Then when passing through the formatFloatValue() method, will return value = 100.00. Look at the following code: https://github.com/onOffice-Web-Org/oo-wp-plugin/blob/25f5a3fee9115f1fb0e05f00762c64b86b6c41ed/plugin/Controller/InputVariableReaderFormatter.php#L55
-> The value displayed in the input search price field in frontend = 100.00

- If I search again the price
->The current display value above input = 100.00, but after going through the parseFloat() method it is replaced "."  to  "". So will return value = 10000 https://github.com/onOffice-Web-Org/oo-wp-plugin/blob/25f5a3fee9115f1fb0e05f00762c64b86b6c41ed/plugin/Controller/InputVariableReaderParser.php#L111
->Then when going through formatFloatValue() method -> will return value = 10000.00

=> **This is the cause of the bug**

I have fixed it and here is video demo, please review and test this branch again. Thanks!

https://github.com/onOffice-Web-Org/oo-wp-plugin/assets/104892108/034d3cd1-44a1-4c93-b70d-734ce1bf9061